addons/aircraft/functions/fnc_initEjectAction.sqf

```diff
@@ -0,0 +1,44 @@
+/*
+ * Author: Dystopian
+ * Add Eject action to vehicle if needed.
+ *
+ * Arguments:
+ * 0: Vehicle <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * _vehicle call ace_aircraft_fnc_initEjectAction
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_vehicle"];
+
+if (unitIsUAV _vehicle) exitWith {};
```

Comment:
If anyone wonders, this command is right. It reports true for unmanned vehicles, not UAV dummy units like the name implies.